tests/Gitonomy/Git/Tests/AbstractTest.php

```diff
@@ -16,7 +16,7 @@
 
 abstract class AbstractTest extends \PHPUnit_Framework_TestCase
 {
-    const REPOSITORY_URL = 'http://github.com/gitonomy/foobar.git';
+    const REPOSITORY_URL = 'http://github.com/giorgioazzinnaro/foobar.git';
```

Comment:
:-1:
simply raise PR against foobar repo

Reply:
Hi @keradus 

I'm afraid tags can't be submitted for PR. Because this change required to read an annotated tag (there already was one though with no body), the same tag should be placed on the upstream repo
See the [releases](https://github.com/giorgioazzinnaro/foobar/releases) for reference.

Cheers
Giorgio

Reply:
could you then raise new "issue" on original repo describing steps to do required steps ?

Reply:
I can't merge this until this is resolved.